### PR TITLE
Chore: replace @walletconnect packages with @reown/walletkit

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves

WalletConnect packages are now under a new namespace on npm. See https://docs.reown.com/walletkit/upgrade/from-web3wallet-web

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,7 +66,6 @@
     "@sentry/react": "^7.91.0",
     "@spindl-xyz/attribution-lite": "^1.4.0",
     "@walletconnect/utils": "^2.17.2",
```
Comment (parius) [inline package.json:68]: could this be also taken from reown?

Comment (katspaugh) [reply package.json:68]: I've asked them about this.

Comment (usame-algan) [reply package.json:68]: Looks like they still recommend this package according to their docs: https://docs.reown.com/walletkit/web/installation

**Review by usame-algan — APPROVED** [review]
👍

File: package.json (hunk 1 above)
Comment (usame-algan) [reply package.json:68]: Although I would suggest to add an explicit dependency to `@walletconnect/core` since we are using it as well.

**Comment by katspaugh** [comment]
I've tested that WalletConnect still works. ✅ 

Screenshot 2024-11-22 at 11 42 43

**Pull request merged**